### DISCUSSION TITLE
v2.0.0 update gives system freeze after logon

### DISCUSSION THREAD

**Discussion by ghost** [ALTaleX531/TranslucentFlyouts#57] [General]
system freezes for a second after activating the context menu everytime I log on. It happens only once until I restart the pc again.

This event might have been triggered by it but I am not sure.
Faulting application name: Explorer.EXE, version: 10.0.22000.2003, time stamp: 0xfbe8ef2a
Faulting module name: SHELL32.dll, version: 10.0.21996.1, time stamp: 0xda78cb7a
Exception code: 0xc0000005
Fault offset: 0x00000000004f76d2
Faulting process id: 0x1bd4
Faulting application start time: 0x01d9d753bc91134d
Faulting application path: C:\Windows\Explorer.EXE
Faulting module path: C:\Windows\System32\SHELL32.dll
Report Id: d56a047f-8fda-42f2-a5e2-607c96fc4528
Faulting package full name: 
Faulting package-relative application ID:

**Comment by ALTaleX531** [comment]
Moved to #58
